cmake/FindBlosc.cmake

```diff
@@ -231,6 +234,19 @@ if(Blosc_FOUND)
       INTERFACE_COMPILE_OPTIONS "${PC_Blosc_CFLAGS_OTHER}"
       INTERFACE_INCLUDE_DIRECTORIES "${Blosc_INCLUDE_DIRS}"
     )
+
+    # Blosc may optionally be compiled with external sources for
+    # lz4, snappy, zlib and zstd. Add them as interface libs if
+    # requested (there doesn't seem to be a way to figure this
+    # out automatically). Note that it's assumed these deps have
+    # been statically built if BLOSC_USE_STATIC_LIBS is ON so that
+    # zstd infers the correct name
+    if(BLOSC_USE_EXTERNAL_SOURCES)
+      set_target_properties(Blosc::blosc PROPERTIES
+        INTERFACE_LINK_DIRECTORIES "${Blosc_LIBRARY_DIRS}"
+        INTERFACE_LINK_LIBRARIES "lz4;snappy;zlib;zstd"
+      )
+    endif()
```

Comment:
No easy fix for this - vcpkg for some reason builds blosc against external deps. blosc itself can be built with any configuration of internal/external sources. afaik, blosc 1.5 does not install a config file. This is only required for vcpkg x64-windows-static variants but must be manually provided (see the new CI test)